database/seeds/ProjectCommentsTableSeeder.php

```diff
@@ -17,15 +17,17 @@ public function run()
         factory(ProjectComment::class, 1)->create();
 
         //Creates project comments, users, and projects
-        /*$faker = Faker\Factory::create();
+        /*$dir = "public/images/seeds/";
```

Comment:
Ah. It could be worth making that a separate seeder/command rather than a commented out section, more user friendly.